### PR TITLE
added entitylist call per workspace / supported TSV uploading from UI

### PR DESCRIPTION
**Pull request by MatthewBemis**
the important part of this PR is the support for TSV uploading, but I also removed the hard-coded listentities call so it's now per workspace which will be nice to have for the blueteam demo

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/page/import_data.cljs [hunk 1]
```diff
@@ -47,29 +47,8 @@
...
-                          [(entity "entityName")
-                           (entity "entityType")
-                           (str (entity "succeeded"))
-                           (entity "message")])
-                     (:uploadResults @state))})]])])})
+        [:div {:style {:paddingTop "22px"}}
+         [:div {:style {:fontSize "125%" :fontWeight 500}} "Import Results" [:br]
+          [:div {:style {:fontSize "75%" :fontWeight 100}} (:message @state)]
+
+          ]])])})
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/import_data.cljs]: Nitpick: merge lines 52-54

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_data.cljs [hunk 2]
```diff
@@ -13,6 +13,14 @@
     :aliquot (filter (fn [entity] (= "aliquot" (entity "entityType"))) entities)
     entities))
 
+(defn- create-mock-entities []
+  (map
+    (fn [i]
+      {:entityType (rand-nth ["sample"])
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_data.cljs]: rand-nth with one item is kinda silly.  Are there other types that we can throw in for mock?

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_data.cljs [hunk 3]
```diff
@@ -57,8 +65,25 @@
...
                                         (make-button "All" :all)]}]
-        [:div {:style {:padding "2em 0"}} [EntitiesList {:entities (:entities props)
-                                                         :active-filter (:active-filter @state)}]]]))})
 
-(defn render-workspace-data [entities]
-  [WorkspaceData {:entities entities}])
+        [:div {:style {:padding "2em 0"}}
+         (cond
+           (:entities-loaded? @state) [EntitiesList {:entities (:entities @state)
+                                                                               :parent-state state}]
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_data.cljs]: Maybe there was a bad merge here at some point?  EntitiesList no longer takes a parent-state property, and should take {:active-filter (:active-filter @state)} instead.

**Comment by MattPutnam** [comment]
Just a few small things, really.  Otherwise :+1:

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_data.cljs (hunk 3 above)
Comment (MatthewBemis) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_data.cljs]: awesome, that explains why the filtering broke! thanks for catching that

**Pull request merged**